feat(hubspot): Add properties support on hubspot source plugin (#10376)
cloudquery/cloudquery: website/pages/docs/plugins/sources/hubspot/overview.mdx

The following example sets up the HubSpot plugin, and connects it to a postgresql destination:

<Configuration/>

### HubSpot Spec

This is the specs that can be used by the HubSpot source Plugin.

- `max_requests_per_second` (`int`, optional. Default: `5`)

  Rate limit per second for requests done HubSpot API, this will depend on your HubSpot plan (https://developers.hubspot.com/docs/api/usage-details#rate-limits)

> - `table_options` ([map[string][TableOptions]](#table-options) spec, optional. Default: `empty`)
> 
>   Table Options for HubSpot entities that will be in sync.
> 
> ### Table Options
> 
> - `table_name` ([TableOptionsSpec](#table-options-spec) spec, optional. Default: `empty`)
> 
>   Table name to define the options when collecting data from the entity on HubSpot.
> 
> ### Table Options Spec
> 
> - `properties` (`[]string`, optional. Default: `[]`)
> 
>   Additional properties to be retrieved from HubSpot when syncing the table entity

Reviewer: ```suggestion
- `table_options` (map[string][TableOptions](#table-options) spec, optional. Default: `empty`)

  Table Options for HubSpot entities that will be synced.

### Table Options

- `properties` (`[]string`, optional. Default: empty)

  Additional properties to be retrieved from HubSpot when syncing the table entity
```

Reviewer: no need for extra section about table options spec. map[string]TableOptions + section about TableOptions is fine
